package.json

```diff
@@ -26,8 +24,18 @@
     "url": "https://github.com/ckeditor/ckeditor4-presets/issues"
   },
   "homepage": "https://ckeditor.com",
+  "scripts": {
+    "install": "napa",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
   "dependencies": {
     "ckeditor4-plugin-exportpdf": "1.0.0"
   },
-  "private": true
+  "devDependencies": {
+    "napa": "^3.0.0"
+  },
+  "napa": {
+    "ckeditor-plugin-scayt": "https://github.com/WebSpellChecker/ckeditor-plugin-scayt#v5.6.4.0",
+    "ckeditor-plugin-wsc": "https://github.com/WebSpellChecker/ckeditor-plugin-wsc#v5.6.4.0"
+  }
```

Comment:
I kind of had higher hopes for this package. The origins of this issue was consistency and we target to have just one method of fetching external plugins. The fact that it requires separate `script` which needs to be run with `npm run install` and packages are in separate list (`napa` entry instead of `dependencies`) makes it again a second approach to installing external plugins. So we exchanged git submodules to `napa` thing...

I though the napa integration will be more seamless/transparent TBH. That being said, it seems to simplify external plugins handling we will need to ask WSC team to add `package.json` to SCAYT/WSC repos. I will report related issues to SCAYT/WSC repos then and when it's ready wa can move forward with this PR.